swagger.yml

```diff
@@ -1591,12 +1667,22 @@ definitions:
         type: integer
       paused:
         type: boolean
+      pitEnabled:
```

Comment:
I don't see this in any doc. Can you link me if I'm missing something?

Reply:
it appears as though it is undocumented! I created it after examining the raw output of the API